samples/Owin.Nowin.HelloWorld/project.json

```diff
@@ -9,11 +9,7 @@
   "compilationOptions": {
```

Comment:
There seems to be some issues with websockets on iis but I'll merge this and then investigate.